tests/platform_tests/test_engine.py

```diff
@@ -101,7 +106,9 @@ def test_create_widget(self):
         Ensures that the _create_widget method is exception safe.
         """
 
-        class _test_widget(sgtk.platform.qt.QtGui.QWidget):
+        from tank.authentication.ui.qt_abstraction import QtGui
```

Comment:
Same here maybe it would be good to explain why we are not using the standard method for importing Qt in the test. (I think there are a few other places as well.)